.gitlab/packaging/rpm.yml

```diff
@@ -92,6 +93,31 @@ installer_rpm-amd64:
     # explicitly disable the check
     PACKAGE_REQUIRED_FILES_LIST: ""
 
+agent_rpm-x64-a7-fips:
+  extends: [.package_rpm_common, .package_rpm_agent_7, .package_rpm_x86]
+  tags: ["arch:amd64"]
+  needs: ["datadog-agent-7-x64-fips"]
+  variables:
+    DD_PROJECT: agent
```

Comment:
I think you'll need to provide something like `OMNIBUS_EXTRA_ARGS: --flavor=fips`, otherwise the package will have the same name as the regular agent package: https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/721204621#L281

Reply:
good catch! we have this in deb but not the rpm packaging

Reply:
fixed in https://github.com/DataDog/datadog-agent/pull/31004/commits/8a52d52c113056e5fad1f3d4bfc4968b66b25d2c